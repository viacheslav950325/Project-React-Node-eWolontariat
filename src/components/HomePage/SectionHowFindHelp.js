import { Box, Typography,Button } from "@material-ui/core";
import PlanningImg from '../../assets/img/planning.svg';

const SectionHowFindHelp = () => {
    return(
        <Box id={"section-how-find-help"}
            display={"flex:"}
            height = {"100%"}
            justifyContent={"center"}
            alignItems={"center"}
        >
            <Typography variant="h1">Jak znaleźć pomoc na pomocny.pl</Typography>
            <Typography variant="body1" paragraph={true}>
                    Pomocny.pl to nie tylko strona z ogłoszeniami dotyczącymi wolontariatu. To miejsce spotkań ludzi, którzy potrzebują przestrzeni 
                    na rozmowę o tym jak można im pomóc. Jeżeli jesteś w trudnej sytuacji i jest coś, w czym można Cię wesprzeć, możesz stworzyć nowe 
                    zadanie lub odezwać się bezpośrednio do któregoś z wolontariuszy. 
                </Typography>
            
            <Box display={'flex'} padding={'4rem'} justifyContent={'space-evenly'} mr={'2rem'}> 
                <Box component="img"  maxHeight={'400px'} alt="" src={PlanningImg}/>
                <Box alignSelf={'center'}>
                    <Typography variant={'h3'}>Krok 1.</Typography>
                    <Typography variant={'subtitle1'} paragraph={true}>
                        Aby rozpocząć kliknij przycisk Stwórz zadanie.
                    </Typography>
                    <Button variant="contained" size={'medium'} color='primary' href={"/"}>Stwórz zadanie</Button>
                    <Typography variant={'h3'}>Krok 2.</Typography>
                    <Typography variant={'subtitle1'} paragraph={true}>
                        Zostaniesz przekierowany do formularza, który poprowadzi Cię dalej. W odpowiednich miejscach wpisz najważniejsze informacje 
                        dotyczące tworzonego przez Ciebie zadania: krótki opis tego, czego potrzebujesz oraz ilu wolontariuszy może się w to zaangażować. 
                        Możesz dodać również zdjęcie.
                    </Typography>
                    <Typography variant={'h3'}>Krok 3.</Typography>
                    <Typography variant={'subtitle1'} paragraph={true}>
                        Kiedy wolontariusz zgłosi się do Twojej akcji, zostaniesz o tym powiadomion_ mailowo.
                    </Typography>
                </Box>
            </Box>
                
        </Box>
    )
}

export default SectionHowFindHelp;