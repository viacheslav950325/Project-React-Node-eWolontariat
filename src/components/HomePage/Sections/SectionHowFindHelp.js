import { Box, Typography,Button } from "@material-ui/core";
import PlanningImg from '../../../assets/img/planning.svg';
import { styled } from '@mui/material/styles';

const StyledBox = styled(Box)(({ theme }) => ({
    display: "flex",
    flexDirection: "column",
    flexWrap: "wrap",
    justifyContent: "center",
    alignItems: "center",
    padding: "2rem 6rem",

    [theme.breakpoints.down('md')]: {
        padding: "2rem",
    },
  }));

  const StyledBoxRow = styled(Box)(({ theme }) => ({
    display: 'flex',
    flexDirection: "row",
    justifyContent: 'space-evenly',
    [theme.breakpoints.down('md')]: {
        flexDirection: 'column',
        flexWrap: 'wrap'
    },
  }));

  const StyledTypography = styled(Typography)(({ theme }) => ({
    [theme.breakpoints.down('md')]: {
        padding: "0",
        margin: '0',
        fontSize: '1rem'
    },
  }));

  const StyledTitle = styled(Typography)(({ theme }) => ({
    [theme.breakpoints.down('md')]: {
        fontSize: '1.6rem',
        lineHeight: "1.4"
    },
  }));


const SectionHowFindHelp = () => {
    return(
        <Box id={"section-how-find-help"}>
            <StyledBox>
                    <StyledTitle  variant="h1">Jak znaleźć pomoc na pomocny.pl</StyledTitle>
                    <StyledTypography variant="body1" paragraph={true} align={'justify'}>
                            Pomocny.pl to nie tylko strona z ogłoszeniami dotyczącymi wolontariatu. To miejsce spotkań ludzi, którzy potrzebują przestrzeni 
                            na rozmowę o tym jak można im pomóc. Jeżeli jesteś w trudnej sytuacji i jest coś, w czym można Cię wesprzeć, możesz stworzyć nowe 
                            zadanie lub odezwać się bezpośrednio do któregoś z wolontariuszy. 
                    </StyledTypography>
            <StyledBox/>
            <StyledBoxRow > 
                    <Box component="img"  maxHeight={'360px'} alt="" src={PlanningImg}/>
                    <Box alignSelf={'center'}>
                        <Box m={3}>
                            <StyledTypography style={{fontWeight:'700'}}>Krok 1.</StyledTypography>
                            <StyledTypography paragraph={true}>
                                Aby rozpocząć kliknij przycisk Stwórz zadanie.
                            </StyledTypography>
                            <Button variant="contained" size={'medium'} color='primary' href={"/TaskForm"}>Stwórz zadanie</Button>
                        </Box>
                        <Box m={3}>
                            <StyledTypography style={{fontWeight:'700'}}>Krok 2.</StyledTypography>
                            <StyledTypography paragraph={true}>
                                Zostaniesz przekierowany do formularza, który poprowadzi Cię dalej. W&nbsp;podpowiednich miejscach wpisz najważniejsze informacje 
                                dotyczące tworzonego przez Ciebie zadania: krótki opis tego, czego potrzebujesz oraz ilu wolontariuszy może się w to zaangażować. 
                                Możesz dodać również zdjęcie.
                            </StyledTypography>
                        </Box>
                        <Box m={3}>
                            <StyledTypography style={{fontWeight:'700'}}>Krok 3.</StyledTypography>
                            <StyledTypography paragraph={true}>
                                Kiedy wolontariusz zgłosi się do Twojej akcji, zostaniesz o tym powiadomion_ mailowo.
                            </StyledTypography>
                        </Box>
                    </Box>
                </StyledBoxRow>
                    
            </StyledBox>
        </Box>
    )
}

export default SectionHowFindHelp;