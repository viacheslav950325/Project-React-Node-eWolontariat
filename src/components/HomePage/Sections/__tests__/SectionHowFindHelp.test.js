import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import SectionHowFindHelp from "../SectionHowFindHelp";

describe("SectionHowFindHelp",()=>{
    it("Should render SectionHowFindHelp correctly",()=>{
        const wrapper = shallow(<SectionHowFindHelp/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});