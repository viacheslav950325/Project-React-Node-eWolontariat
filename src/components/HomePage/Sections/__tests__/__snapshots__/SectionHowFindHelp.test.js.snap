// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHowFindHelp Should render SectionHowFindHelp correctly 1`] = `
<Styled(MuiBox)
  id="section-how-find-help"
>
  <Styled(Styled(MuiBox))>
    <Styled(WithStyles(ForwardRef(Typography)))
      variant="h1"
    >
      Jak znaleźć pomoc na pomocny.pl
    </Styled(WithStyles(ForwardRef(Typography)))>
    <Styled(WithStyles(ForwardRef(Typography)))
      align="justify"
      paragraph={true}
      variant="body1"
    >
      Pomocny.pl to nie tylko strona z ogłoszeniami dotyczącymi wolontariatu. To miejsce spotkań ludzi, którzy potrzebują przestrzeni na rozmowę o tym jak można im pomóc. Jeżeli jesteś w trudnej sytuacji i jest coś, w czym można Cię wesprzeć, możesz stworzyć nowe zadanie lub odezwać się bezpośrednio do któregoś z wolontariuszy.
    </Styled(WithStyles(ForwardRef(Typography)))>
    <Styled(Styled(MuiBox)) />
    <Styled(Styled(MuiBox))>
      <Styled(MuiBox)
        alt=""
        component="img"
        maxHeight="360px"
        src="planning.svg"
      />
      <Styled(MuiBox)
        alignSelf="center"
      >
        <Styled(MuiBox)
          m={3}
        >
          <Styled(WithStyles(ForwardRef(Typography)))
            style={
              Object {
                "fontWeight": "700",
              }
            }
          >
            Krok 1.
          </Styled(WithStyles(ForwardRef(Typography)))>
          <Styled(WithStyles(ForwardRef(Typography)))
            paragraph={true}
          >
            Aby rozpocząć kliknij przycisk Stwórz zadanie.
          </Styled(WithStyles(ForwardRef(Typography)))>
          <WithStyles(ForwardRef(Button))
            color="primary"
            href="/TaskForm"
            size="medium"
            variant="contained"
          >
            Stwórz zadanie
          </WithStyles(ForwardRef(Button))>
        </Styled(MuiBox)>
        <Styled(MuiBox)
          m={3}
        >
          <Styled(WithStyles(ForwardRef(Typography)))
            style={
              Object {
                "fontWeight": "700",
              }
            }
          >
            Krok 2.
          </Styled(WithStyles(ForwardRef(Typography)))>
          <Styled(WithStyles(ForwardRef(Typography)))
            paragraph={true}
          >
            Zostaniesz przekierowany do formularza, który poprowadzi Cię dalej. W podpowiednich miejscach wpisz najważniejsze informacje dotyczące tworzonego przez Ciebie zadania: krótki opis tego, czego potrzebujesz oraz ilu wolontariuszy może się w to zaangażować. Możesz dodać również zdjęcie.
          </Styled(WithStyles(ForwardRef(Typography)))>
        </Styled(MuiBox)>
        <Styled(MuiBox)
          m={3}
        >
          <Styled(WithStyles(ForwardRef(Typography)))
            style={
              Object {
                "fontWeight": "700",
              }
            }
          >
            Krok 3.
          </Styled(WithStyles(ForwardRef(Typography)))>
          <Styled(WithStyles(ForwardRef(Typography)))
            paragraph={true}
          >
            Kiedy wolontariusz zgłosi się do Twojej akcji, zostaniesz o tym powiadomion_ mailowo.
          </Styled(WithStyles(ForwardRef(Typography)))>
        </Styled(MuiBox)>
      </Styled(MuiBox)>
    </Styled(Styled(MuiBox))>
  </Styled(Styled(MuiBox))>
</Styled(MuiBox)>
`;
